omdata/orca/calc.py

```diff
@@ -16,9 +16,10 @@
     "NOSOSCF",
     "NormalConv",
     "DEFGRID2",
+    "ALLPOP"
 ]
 ORCA_SIMPLE_INPUT_QUACC_IGNORE = []
-ORCA_BLOCKS = ["%scf Convergence Tight maxiter 500 end"]
+ORCA_BLOCKS = ["%scf Convergence Tight maxiter 500 end","%elprop Dipole true Quadrupole true end"]
 ORCA_ASE_SIMPLE_INPUT = " ".join([ORCA_FUNCTIONAL] + [ORCA_BASIS] + ORCA_SIMPLE_INPUT)
 OPT_PARAMETERS = {
     "optimizer": Sella,
```

Comment:
While you're at it can you update fmax here to 0.05 per our discussion last week.